src/components/layout/header/auth-panel/auth-panel.tsx

```diff
@@ -69,11 +70,23 @@ export default function AuthPanel({
       variant="text"
     >
       {isValid ? (
-        <Button onClick={() => void handleSignOut()} sx={{ px: 2 }}>
-          <Typography color="var(--foreground)">
-            {t('buttons.sign-out')}
-          </Typography>
-        </Button>
+        <>
+          <MuiLink
+            href="/"
+            color="inherit"
+            component={Link}
+            underline={pathname === '/' ? 'always' : 'hover'}
+          >
+            <Typography fontWeight="normal" variant="h6" sx={{ flexGrow: 1 }}>
+              {t('buttons.main')}
+            </Typography>
+          </MuiLink>
```

Comment:
Great that you've added a main page link 👍

Consider a small improvement, this will make a link look like other nav buttons and also close sidebar on click:
```suggestion
          <Button
            onClick={closeSidebar}
            sx={{ px: 2 }}
            href="/"
            LinkComponent={Link}
          >
            <Typography
              sx={{
                textDecoration: pathname === '/' ? 'underline' : 'none',
              }}
              color="var(--foreground)"
            >
              {t('buttons.main')}
            </Typography>
          </Button>
```

Reply:
You're right, it looks better. I fixed